src/SFA.DAS.LevyTransferMatching.Api/Models/ApplicationRejectRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Api.Models
+{
+    public class ApplicationRejectRequest
+    {
+        public List<string> ApplicationsToReject { get; set; }
```

Comment:
Why is this a list of strings? I'd have thought a list of integers would be sufficient

Reply:
I have refactored it and changed it to list in integers.